frameworks/elastic/src/main/dist/security-pass.sh.mustache

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+{{#SECURITY_ENABLED}}
+ELASTIC_USER={{ELASTICSEARCH_HEALTH_USER}}
+ELASTIC_PASS={{ELASTICSEARCH_HEALTH_USER_PASSWORD}}
+PROTOCOL={{ELASTICSEARCH_HTTP_PROTOCOL}}
+VERSION={{ELASTIC_VERSION}}
```

Comment:
From a security perspective, I don't think this is the way to go, if you look at the generated file you'll see something like this in the sandbox:

```
ELASTIC_USER=elastic
ELASTIC_PASS=changeme
PROTOCOL=http
VERSION=7.6.0
```

The `ELASTIC_PASS` especially is **not** what we want here. We especially go out of the way to prevent this sort of stuff from being printed out in console.

Is there any reason why we can't just use the original variables here as opposed to reassigning them? 
i.e use the original 
- `ELASTICSEARCH_HEALTH_USER`
- `ELASTICSEARCH_HEALTH_USER_PASSWORD`
- `ELASTICSEARCH_HTTP_PROTOCOL`
- `ELASTIC_VERSION`

If the new variables help from an understanding perspective, we should leave a comment saying what each of the env-vars is used for, but I'd rather not see any security leaks here.

Reply:
Will make the password as secret so that it won't be visible even after rendering.